less/sortableTable.less

```diff
@@ -15,11 +15,6 @@ table.sort {
 }
 
 table.sortableTable {
-  width: 100%;
-  margin-bottom: 40px;
-  cursor: default;
-  border-spacing: 0;
```

Comment:
the 3 properties above were replaced with https://github.com/brave/browser-laptop/pull/10265/files#diff-159ff36704e059402c3f17fb8a35a6fcR481.